governance/community_roles.md

```diff
@@ -0,0 +1,34 @@
+# Community roles
+
+This file lays out the key roles within the UK TRE Community, including who currently holds them, their responsibilities, and how these roles are decided.
+
+## Current positions
+
+**Co-chairs**:
+Lucy Cheesman
+*Research Data Services Manager, University of Sheffield*
+
+Madalyn Hardaker
+*Head of eResearch Data Governance, King’s College London*
+
+Simon Li
+*Software and Operations Engineer, University of Dundee*
+
+David Sarmiento Perez
+*Research Project Manager, The Alan Turing Institute*
+
+Hari Sood
+*Research Application Manager, Alan Turing Institute*
+
+Balint Stewart
+*Programme Manager, DARE UK*
+
+## Responsibilities
+### Co-chairs
+* Management of the [UK TRE Community mailbox](mailto:uktrecommunity@gmail.com)
+* Management of the UK TRE Community [GitHub organisation](https://github.com/uk-tre)
+* Management of the UK TRE Community [Google Drive](https://drive.google.com/drive/folders/1yH05sKEZxvHqQwjEkUDIsQdPDMZxY_g2?usp=sharing)
```

Comment:
If this is private only then I think it's better not to link to it, and instead say `Management of the UK TRE Community internal Google Drive`